lib/mpl_toolkits/axisartist/grid_helper_curvelinear.py

```diff
@@ -233,8 +234,8 @@ def f1():
             tick_to_axes = self.get_tick_transform(axes) - axes.transAxes
             for x, y, d, d2, lab in zip(xx1, yy1, dd, dd2, labels):
                 c2 = tick_to_axes.transform((x, y))
-                delta = 0.00001
-                if 0-delta <= c2[0] <= 1+delta and 0-delta <= c2[1] <= 1+delta:
+                if all(mpl.transforms._interval_contains_close((0, 1), c2[i])
+                       for i in range(2)):
```

Comment:
Do we need the index?
```suggestion
                if all(mpl.transforms._interval_contains_close((0, 1), c2)
                       for c2 in tick_to_axes.transform((x, y))):
```

Reply:
I tried a different form instead...